### PR TITLE
Add api instance group with cloud_controller_ng

### PR DESCRIPTION
**Pull request by f0rmiga**
## Description

Bumped the default stemcell version and added the api instance group with the cloud_controller_ng job and the binary buildpack.

## Test plan

Deploy the cf-operator and on SCF:

```
bazel build //deploy/helm/scf:chart && helm template --name scf bazel-bin/deploy/helm/scf/scf-3.0.0.tgz --namespace scf --set "system_domain=$(minikube ip).xip.io" | kubectl apply -f - --namespace scf
```

### CODE REVIEW

File: deploy/helm/scf/assets/operations/instance_groups/api.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Selectively remove jobs temporarily.
```
Comment (andreas-kupries) [inline deploy/helm/scf/assets/operations/instance_groups/api.yaml]: So the instance group is added, but without any jobs actually running ?

Comment (andreas-kupries) [reply deploy/helm/scf/assets/operations/instance_groups/api.yaml]: Never mind, should have read the ticket. This is about adding the CC job in the api group.

Comment (f0rmiga) [reply deploy/helm/scf/assets/operations/instance_groups/api.yaml]: The title says "cloud_controller_ng".

Comment (andreas-kupries) [reply deploy/helm/scf/assets/operations/instance_groups/api.yaml]: Huh ... Oh, PR title, right. I was looking at JIRA ticket title, and then description.

Comment (f0rmiga) [reply deploy/helm/scf/assets/operations/instance_groups/api.yaml]: I'll update JIRA!

**Pull request merged**